[EDU-3741] feat: add http/https guide (#580)
aziontech/docs: src/content/docs/en/pages/observe-journey/data-streaming/integrations/standard-http-https-post.mdx

---

> title: How to use a HTTP/HTTPS POST connector to receive data from Data Streaming

Reviewer: ```suggestion
title: How to use an HTTP/HTTPS POST connector to receive data from Data Streaming
```

description: Hands-on to configure a HTTP/HTTPS POST connector to receive data from Data Streaming.
meta_tags: data streaming, connector, endpoint, http, https, post
namespace: docs_guides_standard_http_htpps_post
permalink: /documentation/products/guides/use-data-streaming/
---

While configuring [data streamings](/en/documentation/products/guides/use-data-streaming/), you need to set up a specific endpoint to stream your Azion data.

> You can use the **Standard HTTP/HTTPS POST** option to configure a streaming with a third-party platform that isn't available as a Data Streaming connector that can receive data via HTTP or HTTPS POST requests.

Reviewer: ```suggestion
You can use the **Standard HTTP/HTTPS POST** option to configure a streaming with a third-party platform that isn't available as a Data Streaming connector. Then, this connector can receive data via HTTP or HTTPS POST requests.
```

---

> ## Configuring the endpoint in Data Streaming
> 
> ### Via RTM

Reviewer: ```suggestion
## Via RTM
```

Reviewer: está explicando uma tarefa só

Author: era pra manter o padrão dos outros guias que falam dos conectores

> Follow these steps to configure the endpointyou in your Azion **Data Streaming**.

Reviewer: ```suggestion
Follow these steps to configure your Azion **Data Streaming** endpoint.
```

> 

Reviewer: não tem os passos para entrar no produto?

In the **Destination** configurations:

1. On the **Endpoint Type** dropdown menu, select **Standard HTTP/HTTPS POST**.
2. On **Endpoint URL**, add the URL of the platform that'll receive the data. Example: `http://myhost.com`

> 3. On **Custom Headers** (*optional*), you can provide additional data the third-party platform accepts or requires, such as an acces key. You can pass up to 5 headers.

Reviewer: ```suggestion
3. On **Custom Headers** (*optional*), you can provide additional data the third-party platform accepts or requires, such as an access key.
- You can pass up to *5 headers*.
```

4. Click the **Save** button.

After saving the configurations, your data will be streamed to the newly configured endpoint.

:::tip
With this type of connector, you can also [set up a payload](/en/documentation/products/guides/observe/data-streaming-set-payload/).
:::

You can keep track of the calls made by Data Streaming to Datadog on [Real-Time Events](/en/documentation/products/real-time-events/). To do so, select **Data Source** > **Data Streaming** and choose the filters options as you wish.

> 
> ### Via API

Reviewer: ```suggestion

---

## Via API
```

Reviewer: aceitando a sugestão no outro subtópico

---
title: How to use an HTTP/HTTPS POST connector to receive data from Data Streaming
description: Hands-on to configure a HTTP/HTTPS POST connector to receive data from Data Streaming.
meta_tags: data streaming, connector, endpoint, http, https, post
namespace: docs_guides_standard_http_htpps_post
permalink: /documentation/products/guides/use-data-streaming/
---

While configuring [data streamings](/en/documentation/products/guides/use-data-streaming/), you need to set up a specific endpoint to stream your Azion data.

You can use the **Standard HTTP/HTTPS POST** option to configure a streaming with a third-party platform that isn't available as a Data Streaming connector that can receive data via HTTP or HTTPS POST requests.

---

## Configuring the endpoint in Data Streaming

### Via RTM

> Follow these steps to configure your Azion **Data Streaming** endpoint.

Author: ```suggestion
You can find detailed steps for the entire configuration on the [How to configure Data Streaming main settings](/en/documentation/products/guides/use-data-streaming/) guide.

Follow the next steps to configure your Azion **Data Streaming** endpoint.
```
